lib/record_store.rb

```diff
@@ -46,7 +46,10 @@ def secrets_path
     end
 
     def zones_path
-      @zones_path ||= Pathname.new(File.expand_path(config.fetch('zones_path'), File.dirname(config_path))).realpath.to_s
+      @zones_path ||= Pathname.new(
+        File.expand_path(config.fetch('zones_path'),
+        File.dirname(config_path)),
+        ).realpath.to_s
```

Comment:
Isn't it going to complain about the fact that the closing parenthesis is not aligned with the `@zones_path` value?

Reply:
It's not complaining after aligned the bottom line with first line.
```
      @zones_path ||= Pathname.new(
        File.expand_path(config.fetch('zones_path'),
        File.dirname(config_path)),
      ).realpath.to_s
```